closure-typing/calculator.js

```diff
@@ -1,5 +1,26 @@
-function calculator() {
-  // your code here
+function calculator(inputExp) {
+
+    let reDecimal = new RegExp(/\d+(\.\d{0,2})?/);
+
+    if (reDecimal.test(inputExp)) {
```

Comment:
What was this check about? If I am right it will check if there somewhere decimal number in the string,
123123.23aaaa is correct in your case, be carefull with RegExp, I should always be sure what they do